dist/crud.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
```

Comment:
Please remove dist folder in commit and add it in gitignore file

Reply:
acknowledge .